pelican/plugins/image_process/image_process.py

```diff
@@ -703,17 +681,34 @@ def process_picture(soup, img, group, settings, derivative):
             img.insert_before(s["element"])
 
 
+def try_open_image(path):
+    try:
+        i = Image.open(path)
+    except UnidentifiedImageError:
+        logger.warning(
+            "%s Source image %s is not supported by Pillow.",
+            LOG_PREFIX,
+            path,
+        )
+        return None
```

Comment:
As I note below, it seems to me that it would be more Python-ic to re-raise the exemption, rather than returning a placeholder value like this.

Something like:

```
try:
    # ...
except UnidentifiedImageError as e:
    # log message
    raise e
```